vite.config.js

```diff
@@ -0,0 +1,19 @@
+import path from 'path';
+
+import { sveltekit } from '@sveltejs/kit/vite';
+
+/** @type {import('vite').UserConfig} */
+const config = {
+  plugins: [sveltekit()],
+  resolve: {
+    alias: {
+      $components: path.resolve('./src/components'),
+      $context: path.resolve('./src/context'),
+      $lib: path.resolve('./src/lib'),
+      $routes: path.resolve('./src/routes'),
+      $styles: path.resolve('./src/styles'),
+    },
```

Comment:
I'm not sure if this setup shows you code coverage or not, but I was able to set it up really easily in another project like so: https://github.com/IMLS/estimating-wifi/blob/scaffold-front-end-add-vitest/imls-frontend/vite.config.js#L18

Reply:
> ooh interesting! can you tell me more about why vitest instead of jest? I'm starting another project with vue +vite and just put jest on, but if vitest would be better... I wanna know!

Vitest inherits the Vite config, so you don't have to replicate anything - eg, Babel config, or in this case, Typescript. Also, Vitest natively uses ESM end-to-end, with fewer headaches to get it configured. It's also pretty compatible with Jest, so was pretty easy as a drop-in replacement.